### DISCUSSION TITLE
Replication between 2 physical single node longhorn deployments on RHEL 8.6.

### DISCUSSION THREAD

**Discussion by kedzone** [longhorn/longhorn#5742] [General]
We have 2 physical single node longhorn deployments on RHEL 8.6. Is it possible to replicate volumes across these 2 nodes and if yes what would be the best way? The replication can be asynchronous.

**Comment by innobead** [comment]
So it's independent cluster for each node? Have you tried DR volume? 

cc @c3y1huang

**Reply by kedzone** [reply]
Yes these are independent clusters for each node. No haven't looked at DR volume.

**Reply by innobead** [reply]
https://longhorn.io/docs/1.4.1/snapshots-and-backups/setup-disaster-recovery-volumes/

Basically just use the same backup store for both clusters, so you can restore a volume from a backup created from the source volume in another cluster.

**Reply by kedzone** [reply]
Thanks, will check this out.
